app.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  englishToPigLatin: function(word) {
+    //word verification
+    if (typeof word !== 'string' || Number(word) / 2) {
```

Comment:
Not sure what the second part of this if statement is trying to handle? It's turning the word into a number and dividing by 2? So basically as long as it's a number that's greater than 0?